package.json

```diff
@@ -125,7 +128,7 @@
     }
   },
   "volta": {
-    "node": "10.20.1",
+    "node": "12.22.4",
```

Comment:
!!breaking change!!

Reply:
Understood, but this should likely match a change to our engines support, and CI (reflected in my comment on the change to the ci-build.yml above).